headers/functions/arm9.h

```diff
@@ -431,13 +460,16 @@ void StrncpySimpleNoPad(char* dest, const char* src, uint32_t n);
 int StrncmpSimple(const char* s1, const char* s2, uint32_t n);
 void StrncpySimpleNoPadSafe(char* dest, const char* src, uint32_t n);
 void SpecialStrcpy(char* dest, const char* src);
+void SpecialNStrcpy(char* dest, const char* src, uint32_t n);
```

Comment:
Can we maybe name these better? How about `StrcpyName` and `StrncpyName`?

Reply:
Yes, those are better names. Will make appropriate changes.